src/scale.ts

```diff
@@ -289,6 +293,7 @@ export function scaleTypeSupportProperty(scaleType: ScaleType, propName: string)
       return isContinuousToContinuous(scaleType) || scaleType === 'band' || scaleType === 'point';
     case 'rangeStep':
     case 'padding':
+    case 'interpolate':
```

Comment:
Do you know where should I include interpolate? If I don't have it here, it will fail the test...
<img width="812" alt="screen shot 2017-01-29 at 6 35 59 pm" src="https://cloud.githubusercontent.com/assets/11696585/22410836/035c90da-e652-11e6-9fa2-cb87ab057815.png">